source/ssl/include/ssl.ooc

```diff
@@ -0,0 +1,45 @@
+/* This file is part of magic-sdk, an sdk for the open source programming language magic.
```

Comment:
Yeah, I agree this looks funny now. I though that this file could be a simple wrapper around the raw openssl api, and we could have more convenient higher level classes here in the future. Something like for opengl stuff.